x-pack/plugins/observability/public/pages/alerts/index.tsx

```diff
@@ -55,6 +71,18 @@ function AlertsPage() {
     workflowStatus,
     setWorkflowStatus,
   } = useAlertsPageStateContainer();
+  const {
+    http,
+    notifications: { toasts },
+  } = useKibana().services;
+  const [alertStatsLoading, setAlertStatsLoading] = useState<boolean>(false);
+  const [alertStats, setAlertStats] = useState<AlertStatsState>({
+    isLoading: false,
+    ruleCount: -1,
+    disabled: -1,
```

Comment:
@claudiopro Out of curiosity why do we start the counting from -1 instead of 0? Do I miss something here?

Reply:
These are just a placeholder. The value should always come from the API response. If an error fetching the data occurs, we should probably hide the values altogether. @hbharding  thoughts?